site/write_ups/2023/29-03-23_writeup.md

```diff
@@ -0,0 +1,127 @@
+# Data Ethics Club: [The Tech We Won't Build](https://2022.internethealthreport.org/episodes/the-tech-we-wont-build/)
+<!--Please don't edit the info panel below-->
+
+<!--
+
+```{admonition} What's this? 
+This is summary of Wednesday 28th July's Data Ethics Club discussion, where we spoke and wrote about the New Republic article [The Rise of Private Spies](https://newrepublic.com/article/161913/we-are-bellingcat-spooked-private-investigators) by Charlie Savage.
+The summary was written by Jessica Woodgate and Huw Day, who tried to synthesise everyone's contributions to this document and the discussion. "We" = "someone at Data Ethics Club". 
+Nina Di Cara and Natalie Thurlby helped with the final edit.
+```
+
+-->
+
+
+
+
+## Discussion
+
+### Discussion
+
+## Discussion
+
+### Discussion
+
+##### Q1 What mechanisms or options exist to you in your context (work or otherwise) to resist the development of technologies you morally disagree with?
+
+We loved the idea of individuals calling out poor or unethical research mentioned in the podcast (where Google employees called out the US defence contract Project Maven for [spying on people]( https://www.wired.com/story/the-line-between-big-tech-and-defense-work/)); employees should have power to hold their employers accountable. To counter misinformation in academia, [Elisabeth Bik]( https://twitter.com/MicrobiomDigest?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor) calls out manipulated images and has had hundreds of papers retracted. [Hotlines]( Whistleblowing: list of prescribed people and bodies - GOV.UK (www.gov.uk)) exist for reporting unethical behaviour, however this can be hard to do alone. Forming groups might help to support one another and create grassroots power. Talking to the people you are working with, as was done in the Project Maven example, might be a more effective way of promoting change, rather than just leaving the project and allowing it to go ahead anyway – just without your input.
```

Comment:
Fixed the hotline link :)
```suggestion
We loved the idea of individuals calling out poor or unethical research mentioned in the podcast (where Google employees called out the US defence contract Project Maven for [spying on people]( https://www.wired.com/story/the-line-between-big-tech-and-defense-work/)); employees should have power to hold their employers accountable. To counter misinformation in academia, [Elisabeth Bik]( https://twitter.com/MicrobiomDigest?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor) calls out manipulated images and has had hundreds of papers retracted. [Hotlines](https://www.gov.uk/government/publications/blowing-the-whistle-list-of-prescribed-people-and-bodies--2) exist for reporting unethical behaviour, however this can be hard to do alone. Forming groups might help to support one another and create grassroots power. Talking to the people you are working with, as was done in the Project Maven example, might be a more effective way of promoting change, rather than just leaving the project and allowing it to go ahead anyway – just without your input.
```